pkg/grpc/api_test.go

```diff
@@ -136,6 +140,23 @@ func startTestServer(port int) *grpc_go.Server {
 	return server
 }
 
+func startGRPCApiServer(port int, testAPIServer *api) *grpc_go.Server {
+	lis, _ := net.Listen("tcp", fmt.Sprintf(":%d", port))
+
+	server := grpc_go.NewServer()
+	go func() {
+		internalv1pb.RegisterDaprInternalServer(server, testAPIServer)
+		if err := server.Serve(lis); err != nil {
+			panic(err)
+		}
+	}()
+
+	// wait until server starts
+	time.Sleep(maxGRPCServerUptime)
```

Comment:
need to remove this, this will hurt cold start badly.

Reply:
This is setting up the test server for testing. :)

Reply:
ah right, silly me, I thought it was the actual server

Reply:
Actual code is just 3 lines, but I added 85 lines for test...